overcast/runner/__init__.py

```diff
@@ -128,6 +128,99 @@ def get_creds_from_env():
 #    d['cacert'] = os.environ.get('OS_CACERT', None)
     return d
 
+
+class Node(object):
+
+    def __init__(self, name, info, keypair, userdata, runner, record_resource=None):
+        self.record_resource = lambda *args, **kwargs: None
+        self.name      = name
+        self.info      = info
```

Comment:
Don't align the ='s.

See https://www.python.org/dev/peps/pep-0008/ (search for "long_variable").